vcbuild.bat

```diff
@@ -416,6 +416,7 @@ for /d %%F in (test\addons\*) do (
     --nodedir="%cd%"
   if !errorlevel! neq 0 exit /b !errorlevel!
 )
+%node_gyp_exe% rebuild --directory="%~dp0benchmark\misc\function_call" --nodedir="%~dp0."
```

Comment:
I'm not sure why, but the call above (L416) uses `--nodedir="%cd%"`, so I'd rather we keep this consistent.
IMHO simplest solution is to add `benchmark\misc\function_call` to L413:
```cmd
for /d %%F in (test\addons\* benchmark\misc\function_call) do (
```